sdk/ml/azure-ai-ml/azure/ai/ml/operations/_component_operations.py

```diff
@@ -112,24 +112,33 @@ def __init__(
 
     @property
     def _code_operations(self) -> CodeOperations:
```

Comment:
Lets avoid casting here. If you have to #type: ignore to avoid the error, then you can do so. You also could just have them return the parent operations class as a return type annotation.